### ISSUE TITLE
Service url `/_/service` does not work when vhost enabled

### ISSUE THREAD

**Issue by rymsha** [enonic/xp#8884]
http://company.com:8080/any-text/_/service/com.enonic.app.features/myservice

Will work with any `any-text` value.

**Comment by rymsha** [comment]
Actually This is the desired behavior: on pages with 404 status service must be accessible, similar to images.

Question is, @vbradnitski why enabled vhost breaks it?

**Comment by rymsha** [comment]
```
enabled = true

mapping.test.host = orlando
mapping.test.source = /
mapping.test.target = /site/default/draft/features
mapping.test.idProvider.system = default
```
http://test:8080/any-text gives 404 as expected
http://test:8080/any-text/_/service/com.enonic.app.features/httptest gives 200 as expected

```
enabled = false
```

http://localhost:8080/site/default/draft/features/any-text gives 404 as expected
http://localhost:8080/site/default/draft/features/any-text/_/service/com.enonic.app.features/httptest gives 200 as expected
http://localhost:8080/site/default/draft/features/_/service/com.enonic.app.features/httptest gives 200 as expected

I cannot reproduce the issue. Maybe it was fixed in 7.8 ? Try on 7.7

**Comment by vbradnitski** [comment]
can reproduce neither in 7.7 or 7.8
